src/components/filter-input/filter-input.tsx

```diff
@@ -15,8 +15,13 @@ export class DropdownFilter {
   @Prop() search: boolean = false;
   @Prop() filter: boolean = false;
   @State() options: Array<any> = []
+  @Event({ bubbles: false }) selectValue: EventEmitter<string>;
```

Comment:
This event should propably also bubble

Reply:
Why? If it bubbles, the callBack will be invoked twice. Would it not?